pandas/tests/series/methods/test_between.py

```diff
@@ -38,3 +39,12 @@ def test_between_period_values(self):
         result = ser.between(left, right)
         expected = (ser >= left) & (ser <= right)
         tm.assert_series_equal(result, expected)
+
+    def test_between_inclusive_is_boolean_string(self):
```

Comment:
add GH issue number to added test.

To 'lock in' your changes best to ensure tests exist for:

  - [ ] accepting all (inc new) arguments in `validate_endpoints` as well as old ones for backwards compat, probably as two separate tests: and correct boundary functionality.
  - [ ] accepting all (inc new) arguments in `Series.between` probably as as two sep tests, and correct boundary functionality.
  - [ ] making sure the errors are hit with bad input

Reply:
@attack68 Thank you for the feedback, how exactly do I go about adding an issue number to the test case? Also should there be an issue number for each new test added?